go/test/endtoend/vtorc/api/api_test.go

```diff
@@ -134,7 +134,7 @@ func TestAPIEndpoints(t *testing.T) {
 		status, resp, err = utils.MakeAPICall(t, vtorc, "/api/replication-analysis?keyspace=ks&shard=80-")
 		require.NoError(t, err)
 		assert.Equal(t, 200, status, resp)
-		assert.Equal(t, "[]", resp)
+		assert.Equal(t, "null", resp)
```

Comment:
These `[]` to `null` changes are coming because I changed an empty slice to a nil slice.